### ISSUE TITLE
pkg.refresh_db requiring arguments (regression?)

### ISSUE THREAD

**Issue by mudfly** [saltstack/salt#24529]
When executing the pkg.refresh_db module against CentOS 6 minions, I am getting a TypeError that the function requires 3 arguments to be passed in. I have been able to execute this in the past without passing any arguments. The documentation for salt.modules.yumpkg.refresh_db() example shows that no arguments are required.

http://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.yumpkg.html#salt.modules.yumpkg.refresh_db

```
salt *salt* pkg.refresh_db
TypeError encountered executing pkg.refresh_db: refresh_db() takes exactly 3 arguments (0 given). See debug log for more info.
```
```
salt-call pkg.refresh_db
Passed invalid arguments: refresh_db() takes exactly 3 arguments (0 given)
```
```
salt-call --versions-report
           Salt: 2015.5.0
         Python: 2.6.6 (r266:84292, Jan 22 2014, 09:42:36)
         Jinja2: 2.2.1
       M2Crypto: 0.20.2
 msgpack-python: 0.4.6
   msgpack-pure: Not Installed
       pycrypto: 2.0.1
        libnacl: Not Installed
         PyYAML: 3.10
          ioflo: Not Installed
          PyZMQ: 14.3.1
           RAET: Not Installed
            ZMQ: 3.2.5
           Mako: Not Installed
```

**Comment by dmyerscough** [comment]
@mudfly This issue has been fixed by @jfindlay and back ported to 2015.5.

@jfindlay this issue can be closed out

**Comment by jfindlay** [comment]
@mudfly, @dmyerscough is right, this has been fixed in #24088.

**Comment by mudfly** [comment]
@jfindlay @dmyerscough Thanks, looking forward to updated packages!
